wiam-connector/src/main/java/com/sequenceiq/cloudbreak/wiam/client/GrpcWiamClient.java

```diff
@@ -0,0 +1,45 @@
+package com.sequenceiq.cloudbreak.wiam.client;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import com.cloudera.thunderhead.service.workloadiam.WorkloadIamProto.SyncUsersResponse;
+import com.sequenceiq.cloudbreak.auth.crn.RegionAwareInternalCrnGeneratorFactory;
+import com.sequenceiq.cloudbreak.grpc.ManagedChannelWrapper;
+
+import io.opentracing.Tracer;
+
+@Component
+public class GrpcWiamClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GrpcWiamClient.class);
+
+    @Qualifier("wiamManagedChannelWrapper")
+    @Inject
+    private ManagedChannelWrapper channelWrapper;
+
+    @Inject
+    private Tracer tracer;
+
+    @Inject
+    private RegionAwareInternalCrnGeneratorFactory regionAwareInternalCrnGeneratorFactory;
+
+    public void syncUsersInEnvironment(String accountId, String environmentCrn, String requestId) {
+        WiamClient wiamClient = createClient();
+        LOGGER.debug("Initiating usersync for [{}]", environmentCrn);
+        SyncUsersResponse response = wiamClient.syncUsersInEnvironment(Optional.ofNullable(requestId).orElseGet(() -> UUID.randomUUID().toString()),
```

Comment:
I think it's a bit confusing to change the parameter order between GrpcWiamClient and WiamClient. Is there any reason to put the requestId first? If not, then lets use the same parameter order.